src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -342,6 +342,31 @@ public bool TryRemove(TKey key, [MaybeNullWhen(false)] out TValue value)
             return TryRemoveInternal(key, out value, false, default);
         }
 
+        /// <summary>Removes a key and value from the dictionary.</summary>
```

Comment:
@carlossanlop @mairaw we need to port this docs to the api-docs repo. @carlossanlop is it a direct port? or are there any guidelines?

Reply:
As long as the Docs repo has the reference assembly for these APIs, I can do the automatic porting.
I can share the instructions with you to do the automatic port, in case you want to do it yourself. I'll send you the OneNote.